### ISSUE TITLE
Star Wars Racer Clipping Problems

### ISSUE THREAD

**Issue by davidzkm** [mupen64plus/mupen64plus-video-glide64mk2#68]
Hi all,

I tried playing the N64 Star Wars Pod Racer Game.

With Glide64 as core it runs silky smooth. The only problem that occurs is a very significant clipping of everything that comes too close to the camera. This leads to disappearing vehicles etc.

Is there any known workaround for this? Since the game in general runs so well it would be a great showcase of what could be played with a RP2.

I also tried Rice but Glide64 is way more performant plus in Rice some strange flickering occurs in the landscape which is very annoying.

Thanks in advance

**Comment by Krathoon** [comment]
Yeah. I am running into this too. The thing is, this particular emulator runs the best on Emuelec. So, it is a pretty annoying bug.
